datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -1089,6 +1090,24 @@ private void hardDeleteDatastream(Datastream ds, List<Datastream> activeStreams)
     _adapter.deleteDatastream(ds.getName());
   }
 
+  /**
+   * Invokes post datastream state change action of connector for given datastream.
+   * @param datastream the datastream
+   * @throws DatastreamException if fails to perform post datastream action
+   */
+  public void invokePostDataStreamStateChangeAction(final Datastream datastream) throws DatastreamException {
+    _log.info("Invoke post datastream state change action");
+    try {
+      final String connectorName = datastream.getConnectorName();
+      final ConnectorInfo connectorInfo = _connectors.get(connectorName);
+      connectorInfo.getConnector().postDatastreamStateChangeAction(datastream);
+    } catch (DatastreamException e) {
+      _log.error("Failed to perform post datastream state change action datastream={}", datastream);
```

Comment:
Just print the datastream name. In case of MM, the datastream objects are really huge and make the logs noisy. Please check all the logs in the patch and just print the name unless explicitly required.

Reply:
Ok, we print the whole object other places so I was not whether that was mandatory for open source project. refactored.

Reply:
Thanks. We tried to fix as many places as we could. This one clearly can work with just the datastream name.